issues/Issues.md

```diff
@@ -0,0 +1,60 @@
+# Known Issues
+
+## Windows Issues
+
+####/dev/null git apply issue
+
+New version of git is required to solve this issue.
+
+#### CreateProcess issue
+
+```
+echo  Makefile
+CreateProcess(NULL,echo Makefile,...)
+process_begin: CreateProcess(NULL, echo Makefile, ...) failed.
+Putting child 01DD8B10 (_build) PID 31536216 on the chain.
+Live child 01DD8B10 (_build) PID 31536216
+make (e=2): The system cannot find the file specified.
+Reaping losing child 01DD8B10 PID 31536216
+make: *** [Makefile:323: _build] Error 2
+Removing child 01DD8B10 PID 31536216 from chain.
+```
+
+This issue is due to missing `sh` or `bash` in path.
+Adding `sh` or `bash` will fix the issue.
+
+#### Long paths issue
```

Comment:
Could add a reference to this documentation: https://docs.microsoft.com/en-us/windows/desktop/fileio/naming-a-file#paths

Reply:
It has been added.